Content.Server/Fax/FaxMachineComponent.cs

```diff
@@ -0,0 +1,124 @@
+﻿using Content.Shared.Containers.ItemSlots;
+using Robust.Shared.Audio;
+
+namespace Content.Server.Fax;
+
+[RegisterComponent]
+public sealed class FaxMachineComponent : Component
+{
+    /// <summary>
+    /// Name with which the fax will be visible to others on the network
+    /// </summary>
+    [ViewVariables(VVAccess.ReadWrite)]
+    [DataField("name")]
+    public string FaxName { get; set; } = "Unknown";
+
+    /// <summary>
+    /// Device address of fax in network to which data will be send
+    /// </summary>
+    [ViewVariables(VVAccess.ReadWrite)]
+    [DataField("destinationAddress")]
+    public string? DestinationFaxAddress { get; set; }
+
+    /// <summary>
+    /// Contains the item to be send, assumes it's paper...
+    /// </summary>
+    [DataField("paperSlot")]
+    public ItemSlot PaperSlot = new();
```

Comment:
This should probably have `required: true` to ensure yaml prototypes properly set it up